src/test/java/name/webdizz/sonar/grammar/sensor/GrammarViolationTriggerTest.java

```diff
@@ -0,0 +1,66 @@
+package name.webdizz.sonar.grammar.sensor;
+
+import com.swabunga.spell.event.SpellCheckEvent;
+import java.util.Arrays;
+import java.util.List;
+import static org.junit.Assert.*;
+import org.junit.Test;
+import static org.mockito.Mockito.*;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+/**
+ *
+ * @author Oleg_Sopilnyak1
+ */
+public class GrammarViolationTriggerTest {
+    
+    public GrammarViolationTriggerTest() {
+    }
+
+    /**
+     * Test of spellingError method, of class GrammarViolationTrigger.
+     */
+    @Test
+    public void testSpellingError() {

```

Comment:
too big test, it's not clear what we're testing
please split into several small